src/emulator/auth/state.ts

```diff
@@ -153,6 +155,10 @@ export class ProjectState {
       deleteProviders.push(PROVIDER_PASSWORD);
     }
 
+    if (!oldInitialEmail && user.initialEmail) {
```

Comment:
Let's just get rid of the `oldInitialEmail` variable. It really does't hurt to set the key in the map over and over again.

Reply:
Done.